utils/airtable_es2015.js

```diff
@@ -33,13 +34,19 @@ var fetchTableFromAirtable = async function fetchTableFromAirtable(table) {
 
 var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateFromAirtable() {
   let dataStore = {};
-  dataStore.benefits = await fetchTableFromAirtable("benefits");
-  dataStore.eligibilityPaths = await fetchTableFromAirtable("eligibilityPaths");
-  dataStore.needs = await fetchTableFromAirtable("needs");
-  dataStore.examples = await fetchTableFromAirtable("examples");
-  dataStore.translations = await fetchTableFromAirtable("translations");
-  dataStore.areaOffices = await fetchTableFromAirtable("areaOffices");
-  dataStore.timestamp = await Date.now();
+
+  airtableConstants.tableNames.forEach(async function(tableName) {
+    dataStore[tableName] = await fetchTableFromAirtable(tableName);
+  });
+
+  // dataStore.benefits = await fetchTableFromAirtable("benefits");
+  // dataStore.eligibilityPaths = await fetchTableFromAirtable("eligibilityPaths");
+  // dataStore.needs = await fetchTableFromAirtable("needs");
+  // dataStore.examples = await fetchTableFromAirtable("examples");
+  // dataStore.translations = await fetchTableFromAirtable("translations");
+  // dataStore.areaOffices = await fetchTableFromAirtable("areaOffices");
+  // dataStore.timestamp = await Date.now();
```

Comment:
Can this line still go in? We need it for a Log

Reply:
whoops, to many comments. 👍